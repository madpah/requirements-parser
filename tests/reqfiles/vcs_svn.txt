-e svn+svn://svn.myproject.org/svn/MyProject#egg=MyProject
-e svn+http://svn.myproject.org/svn/MyProject/trunk@2019#egg=MyProject

svn+svn://svn.myproject.org/svn/MyProject#egg=MyProject
svn+http://svn.myproject.org/svn/MyProject/trunk@2019#egg=MyProject
