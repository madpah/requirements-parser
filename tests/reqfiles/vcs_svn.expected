[
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": true, 
  "uri": "svn+svn://svn.myproject.org/svn/MyProject", 
  "extras": [], 
  "vcs": "svn", 
  "path": null, 
  "line": "-e svn+svn://svn.myproject.org/svn/MyProject#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": true, 
  "uri": "svn+http://svn.myproject.org/svn/MyProject/trunk", 
  "extras": [], 
  "vcs": "svn", 
  "path": null, 
  "line": "-e svn+http://svn.myproject.org/svn/MyProject/trunk@2019#egg=MyProject", 
  "specs": [], 
  "revision": "2019"
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "svn+svn://svn.myproject.org/svn/MyProject", 
  "extras": [], 
  "vcs": "svn", 
  "path": null, 
  "line": "svn+svn://svn.myproject.org/svn/MyProject#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "svn+http://svn.myproject.org/svn/MyProject/trunk", 
  "extras": [], 
  "vcs": "svn", 
  "path": null, 
  "line": "svn+http://svn.myproject.org/svn/MyProject/trunk@2019#egg=MyProject", 
  "specs": [], 
  "revision": "2019"
 }
]